pkg/kubelet/cm/container_manager.go

```diff
@@ -93,7 +93,8 @@ type ContainerManager interface {
 	// any registered device plugin resource
 	UpdatePluginResources(*schedulernodeinfo.NodeInfo, *lifecycle.PodAdmitAttributes) error
 
-	InternalContainerLifecycle() InternalContainerLifecycle
+	// GetInternalContainerLifecycle returns InternalContainerLifecycle that handles container resource management
+	GetInternalContainerLifecycle() InternalContainerLifecycle
```

Comment:
I'll take the comment, but the rename seems like an unnecessary change?

Reply:
It is not just renaming, it caches the internalLifecycle. internalLifecycle has state(its filed member cpuManager and topologyManager), and we need to use the same internalLifecycle for all the places that uses internalLifecycle, so we need to cache the internalLifecycle.